docs_source_files/content/support_packages/chrome_devtools.de.md

```diff
@@ -123,7 +123,10 @@ With Selenium and devtools integration, you can automate the input of basic auth
 
 {{< code-tab >}}
   {{< code-panel language="java" >}}
-# Please raise a PR to add code sample
+Predicate<URI> uriPredicate = uri -> uri.getHost().contains("your-domain.com");
+
+driver.register(uriPredicate, UsernameAndPassword.of("admin", "password"));
```

Comment:
Is this should not be in this way?

((HasAuthentication) driver).register(uriPredicate, UsernameAndPassword.of("admin", "password"));

Reply:
Hi @JS31096,

Thanks for your comment)
You right, it's not clear that driver should be instance of ChromiumDriver. I've added cast to HasAuthentication.
Please approve)